### DISCUSSION TITLE
Train data question (MLS)

### DISCUSSION THREAD

**Discussion by gen35** [openai/whisper#2220] [General]
Hi, 
To ensure fairness of internal evaluation I can't use "Multilingual LibriSpeech (MLS)". I couldn't use Whisper if MLS is in the training dataset. 

Is it possible to confirm that MLS wasn't used for training?

**Comment by glangford** [comment]
According to the paper, when comparing whisper to prior work, "Whisper performs well on Multilingual LibriSpeech, outperforming XLS-R" ... so it would not have been trained on MLS. 

See 3.4 -
https://arxiv.org/pdf/2212.04356

**Reply by glangford** [reply]
...after the initial release, the large-v2 model was released. In the announcement of large-v2, there isn't a mention of new training data being added:

* https://github.com/openai/whisper/discussions/661

For large-v3, the number of hours of training data increased but MLS is not mentioned

* https://github.com/openai/whisper/discussions/1762
